xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/Translations.ru.xml

```diff
@@ -64,6 +64,17 @@ platform.wiki.users.leave.confirmation=Вы уверены, что хотите
 platform.wiki.users.leave.confirmation.no=Нет
 platform.wiki.users.leave.confirmation.yes=Да
 platform.wiki.users.leave.empty=Покинуть Вики
-platform.wiki.users.leave.title=Покинуть Вики "{0}" 
+platform.wiki.users.leave.title=Покинуть Вики "{0}"
+## Create
+platform.wiki.create.button=Создать Вики
+platform.wiki.create.error=
```

Comment:
@sytkosa Any idea why this is empty (doesn't look correct)? Thanks!

Reply:
Non existing keys are automatically added even if they are empty.

Reply:
@atallahade This is really bad since it means the text will be displayed empty and not fallback on the java properties mechanism AFAIK (i.e getting the English version). It's a big bug I think.

Reply:
> Non existing keys are automatically added even if they are empty.

This is very wrong if that's the case.

Reply:
Is there a case where a translation string can be empty? If not, it would be safer to always use the fallback string when the translation is empty.

Reply:
Well, I found this case: `xe.userdirectory.emptyvalue=`. I can remove empty strings from translated flies (non english) if needed. English fallbacks will then be used when the translation is an empty string.

Reply:
> Is there a case where a translation string can be empty?

Yes there is valid use cases with empty strings.